server/channels/api4/ip_filtering_test.go

```diff
@@ -37,9 +37,11 @@ func Test_getIPFilters(t *testing.T) {
 		ipFiltering := &mocks.IPFilteringInterface{}
 		th.App.Srv().IPFiltering = ipFiltering
 
-		th.App.Srv().RemoveLicense()
+		err := th.App.Srv().RemoveLicense()
```

Comment:
`th.App.Srv().RemoveLicense()` returns a `*model.AppError`, not an `error`. The variable should be named `appErr` to avoid confusion, and the method to check it should be `require.Nil`.